doc/BBCode.md

```diff
@@ -65,17 +65,17 @@ table.bbcodes > * > tr > th {
   <td><a href="http://friendi.ca" target="external-link">Friendica</a></td>
 </tr>
 <tr>
-  <td>[img]https://raw.githubusercontent.com/friendica/friendica/stable/images/friendica-32.jpg[/img]</td>
-  <td><img src="https://raw.githubusercontent.com/friendica/friendica/stable/images/friendica-32.jpg" alt="Immagine/foto"></td>
+  <td>[img=32x32]https://raw.githubusercontent.com/friendica/friendica/stable/images/friendica.svg[/img]</td>
+  <td><img src="https://raw.githubusercontent.com/friendica/friendica/stable/images/friendica.svg" style="width: 32px;" alt="Immagine/foto"></td>
```

Comment:
This breaks the documentation steps for using the `[img]` tag:
- Without any parameters
- With an alt text
- With size parameter
```suggestion
  <td>[img]https://raw.githubusercontent.com/friendica/friendica/stable/images/friendica-32.png[/img]</td>
  <td><img src="https://raw.githubusercontent.com/friendica/friendica/stable/images/friendica-32.png"></td>
```

Reply:
It was in an effort to remove as many PNG references as possible, but I see the issue.